source/project-stories/reed-learning.erb

```diff
@@ -0,0 +1,156 @@
+---
+title: "Project Story - Reed Learning"
+
+header:
+  client: "Reed Learning"
+  introduction: "Leading provider of corporate learning, Reed Learning, bursts into the digital space"
+  image: "/project-stories/reed-learning/header.jpg"
+  image_description: "Reed Learning team meeting"
+
+introduction:
+  title: "Partnering with Reed Learning to kickstart innovation to reinvent their training delivery to one crucial market: Corporates"
+  content:
+    - "Introduced lean and agile product development techniques to internal team"
+    - "Digital service developed internally and in the hands of customers within weeks"
+    - "New model for face-to-face training designed and delivered with excellent feedback"
+  image: "/project-stories/reed-learning/kickstart-meeting.jpg"
+  image_description: "Reed Learning innovation kickstart meeting"
+
+statistics_1:
+  title: "Reed Learning in numbers"
+  number: "3"
+  text: "key customers trialling first digital service"
+
+statistics_2:
+  number: "x/10"
```

Comment:
This seems incorrect, has the copy changed since?
<img width="293" alt="screen shot 2015-12-15 at 10 21 34" src="https://cloud.githubusercontent.com/assets/885223/11808023/a2e8cfb4-a315-11e5-8ccd-0bf0caafc6fd.png">